CONTRIBUTING_GUIDE.md

```diff
@@ -54,6 +54,65 @@ Guidelines to common code contributions:
 | Compiling requirements | Scale 2.13 |
 | Running requirements   | JDK 17     |
 
+> Tips: You can refer the [document](https://www.scala-lang.org/download/2.13.12.html) to install Scale 2.13
+
+## Local Debug with IDEA
+
+### Gradle
+
+Build AutoMQ is the same with Apache Kafka. Kafka uses Gradle as its project management tool. The management of Gradle projects is based on scripts written in Groovy syntax, and within the Kafka project, the main project management configuration is found in the build.gradle file located in the root directory, which serves a similar function to the root POM in Maven projects. Gradle also supports configuring a build.gradle for each module separately, but Kafka does not do this; all modules are managed by the build.gradle file in the root directory.
+
+It is not recommended to manually install Gradle. The gradlew script in the root directory will automatically download Gradle for you, and the version is also specified by the gradlew script.
+
+### Build
+```
+./gradlew jar -x test
+```
+
+### Prepare S3 service
+Refer this [doc](https://docs.localstack.cloud/getting-started/installation/) to install localstack to mock a local s3 service or use AWS S3 service directly. 
+
+If you are using localstack then create a bucket with the following command:
+```
+aws s3api create-bucket --bucket ko3 --endpoint=http://127.0.0.1:4566
+```
+### Modify Configuration
```

Comment:
Whether to override the configuration using --override.

Reply:
@superhx Because this is for developers. Let them know the detailed configuration is helpful. So we use properties file here to start AutoMQ.